site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/widgets/Worker.ts

```diff
@@ -0,0 +1,103 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode, WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+import * as styles from '../styles/worker.css';
```

Comment:
Ok, either way is fine, I think @rishson was going to do that with the other tutorials this week.